test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -4518,6 +4540,207 @@ BOOST_AUTO_TEST_CASE(overwriting_inheritance)
 	BOOST_CHECK(callContractFunction("checkOk()") == encodeArgs(6));
 }
 
+BOOST_AUTO_TEST_CASE(inherited_function_clash)
```

Comment:
I would not say this is a clash, it is just overriding a function, and we should already have tests for that.